### PR TITLE
Mario Party Community Netplay

### PR DESCRIPTION
**Pull request by CrankySupertoon**
sorry my git bash is acting up. i flattened the commit

### CODE REVIEW

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

**Comment by Techjar** [comment]
Lots of possessives are missing apostrophes, for too many for me to specifically point out.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 1]
```diff
@@ -10,10 +10,360 @@
...
+4BFFECC9 38000028
+98030000 3B9C0001
+481E5098 00000000
+041E991C 4BE1AF44
+
+$Hammer Bro Orb Steals 20 Coins
+041AE1E4 3AC00014
+041AEBE0 38A0FFEC
+
+$Mic Minigames have a 10% chance of appearing instead of 20
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP7E01.ini]: Should also say 20% for correctness.

File: Data/Sys/GameSettings/GP7E01.ini [hunk 2]
```diff
@@ -10,10 +10,360 @@
...
+e2000001 80008000
+*Chests now cost 20 Coins, and Bob-omb/Nothing Chests now contain 20 Coins.
+
+$Board - Neon Heights: Bowser Path for Free
+204FDEF4 2C03000A
+044FDEF4 2C030000
+044FDF5C 38A00000
+E2000001 80008000
+
+$Board - Pagoda Peak: Bottle Rocket Launch Always Succeds Pagoda Peak
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP7E01.ini]: Succeeds

File: Data/Sys/GameSettings/GP6E01.ini [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+7C7D1B78 00000000
+
+$Sluggish Shroom Rolls 1-5
+04186608 38000005
+04186b80 4bfffff8
+
+$Minigame - Crate and Peril: Ability to Jump
+0408074c 60fb0003
+
+$Mingame - Dizzy Rotisserie: Lower Starting Time
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP6E01.ini]: Minigame

File: Data/Sys/GameSettings/GP6E01.ini [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+$Minigame - Treasure Trawlers: All Scavenged Items are Regular Chests
+20501374 e3e100a8
+c24fc478 00000003
+2c030000 40810008
+38600000 7c7c1b78
+60000000 00000000
+e2000001 80008000
+
+$Minigame Replacement - Replace Odd Card Out -> What Goes Up...
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP6E01.ini]: These should probably be `➜` like the Mario Party 5 codes, as it looks better.

File: Data/Sys/GameSettings/GP5E01.ini [hunk 5]
```diff
@@ -378,3 +462,411 @@ $Press L+X: Player Stops Moving
...
+38600000 38030001
+60000000 00000000
+E2000001 80008000
+
+$Minigame - Pop Star Piranhas: Halved Time to Pick
+2046983c 3880012c
+0446983c 38800096
+e2000001 80008000
+
+$Minigame Replacment - Get a Rope ➜ Piece Out
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Replacement

File: Data/Sys/GameSettings/GP5E01.ini [hunk 6]
```diff
@@ -378,3 +462,411 @@ $Press L+X: Player Stops Moving
...
+2822a4c4 00000027
+0222a4c4 00000025
+e2000001 80008000
+
+$Minigame Replacement - Replace Rumble Fumble ➜ Fish Upon a Star
+2822a4c4 00000041
+0222a4c4 00000040
+e2000001 80008000
+*
+$Minigame Replacement - Replace Scaldin Cauldron ➜ Rain of Fire
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Scaldin'

File: Data/Sys/GameSettings/GP5E01.ini [hunk 7]
```diff
@@ -378,3 +462,411 @@ $Press L+X: Player Stops Moving
...
+0446EA6C 38800710
+E2000001 80008000
+
+$Minigame -  Coney Island: Bigger Cones Dont Slow You Down
+204763E4 3C608048
+044763E0 38000000
+0447663C 38000000
+E2000001 80008000
+
+$Mingame - Flower Shower: All Flowers Worth 3 Points
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Minigame

File: Data/Sys/GameSettings/GP5E01.ini [hunk 8]
```diff
@@ -378,3 +462,411 @@ $Press L+X: Player Stops Moving
...
+20459BEC 801F003C
+04459BEC 38000003
+E2000001 80008000
+
+$Mingame - Ground Pound Down: No Rocks Until End
```
Comment (Techjar) [inline Data/Sys/GameSettings/GP5E01.ini]: Minigame

File: Data/Sys/GameSettings/RM8E01.ini [hunk 9]
```diff
@@ -0,0 +1,494 @@
...
+de000000 80008180
+301a6ba0 4bffb149
+141a6b8c 48000030
+e0000000 80008000
+
+$Netplay Safe Kill Music
+042ce168 00000001
+0404d9fc 60000000
+
+$Board - Goombas Boody Boardwalk: Remove Dolphin Shuttles
```
Comment (Techjar) [inline Data/Sys/GameSettings/RM8E01.ini]: Goomba's Booty Boardwalk

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

**Comment by Techjar** [comment]
There are still numerous grammar errors in this PR.

**Pull request closed without merging**